WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -127,6 +127,11 @@ public SendViewModel(WalletViewModel walletVm)
 					SubtractFee = amount == _wallet.Coins.TotalAmount() && !(IsFixedAmount || IsPayJoin)
 				};
 
+				if (_coinJoinManager is { } coinJoinManager)
+				{
+					await coinJoinManager.WalletEnteredTxPreviewAsync(_wallet);
```

Comment:
You mean we should just fire and forget it? Or should we make the `WalletEnteredTxPreviewAsync` method sync?

Reply:
This is a kind of a lie. It entered the transaction preview screen when that page appeared. Currently, we are on the Send first screen.

Reply:
Moved the call to its proper place.
[5f876e6](https://github.com/zkSNACKs/WalletWasabi/pull/10500/commits/5f876e661b9c84a8c8450ad05ee580765a344371)

Reply:
Guys this is an indicator for the coinjoinmanager to stop - not a precision battle. I can even put the indicator when somebody entered the amount. 

The consequences of correcting this "lie", lead to serious complexity increase. 
https://github.com/zkSNACKs/WalletWasabi/pull/10500#discussion_r1180377512

I suggest putting it back here - it will be much simpler and the same behavior.

Reply:
@molnard @adamPetho 
In that case, if it is just a sign to stop and you can put it anywhere, then the naming is incorrect. Because it suggests that logic should be executed when the user is on the transaction preview screen. 

If the naming is correct then it should be in OnNavigatedTo() method as the convention is to put logic there that needs to be executed when the dialog appears. (Imagine if later there will be a new dialog between Send and Preview. So the future developer should move this call to the new dialog?) 

Change the method name then... IMO calling a method based on UI terms in the business code is also not good.